### PR TITLE
Minor readability improvements

### PR DESCRIPTION
**Pull request by rakyll**
- Shorter names for variables and types.
- Print flag values instead of pointer values.
- Use -config instead of -exporters-yaml because config contains more than exporters.

### CODE REVIEW

File: cmd/ocagent/config.go [hunk 1]
```diff
@@ -20,26 +20,26 @@ import (
...
 
-type config struct {
+type interceptorConfig struct {
 	// The address to which the OpenCensus interceptor will be bound and run on.
 	Address string `yaml:"address"`
 }
 
-func (tcfg *topLevelConfig) openCensusInterceptorAddressOrDefault() string {
-	if tcfg == nil || tcfg.OpenCensusInterceptorConfig == nil || tcfg.OpenCensusInterceptorConfig.Address == "" {
+func (c *config) interceptorAddress() string {
```
Comment (odeke-em) [inline cmd/ocagent/config.go]: Please revert this, we are going to have more than just the OpenCensus interceptor:
* OpenCensus interceptor
* Zipkin interceptor
* Istio Mixer
* Jaeger
and many more

Please take a look at https://github.com/census-instrumentation/opencensus-service#implementation-details-of-agent-server

File: cmd/ocagent/main.go [hunk 2]
```diff
@@ -37,31 +37,30 @@ import (
...
-	ocInterceptorAddr := ownConfig.openCensusInterceptorAddressOrDefault()
-
+	interceptorAddr := agentConfig.interceptorAddress()
 	traceExporters, _, closeFns := exporterparser.ExportersFromYAMLConfig(yamlBlob)
-
 	commonSpanReceiver := exporter.OCExportersToTraceExporter(traceExporters...)
 
 	// Add other interceptors here as they are implemented
-	ocInterceptorDoneFn, err := runOCInterceptor(ocInterceptorAddr, commonSpanReceiver)
+	interceptorDoneFn, err := runOCInterceptor(interceptorAddr, commonSpanReceiver)
```
Comment (odeke-em) [inline cmd/ocagent/main.go]: Ditto about reverting this. We are going to have a bunch of interceptors not just the OpenCensus interceptor

File: cmd/ocagent/main.go [hunk 3]
```diff
@@ -37,31 +37,30 @@ import (
 )
 
 func main() {
-	exportersYAMLConfigFile := flag.String("exporters-yaml", "config.yaml", "The YAML file with the configurations for the various exporters")
+	var configFile string
```
Comment (odeke-em) [inline cmd/ocagent/main.go]: It'd be nice to keep the YAML in the name to aid in readability.

File: cmd/ocagent/main.go [hunk 4]
```diff
@@ -37,31 +37,30 @@ import (
...
+		log.Fatalf("Cannot read the YAML file %v error: %v", configFile, err)
 	}
-	ownConfig, err := parseOCAgentConfig(yamlBlob)
+	agentConfig, err := parseOCAgentConfig(yamlBlob)
 	if err != nil {
-		log.Fatalf("Failed to parse own configuration %v error: %v", exportersYAMLConfigFile, err)
+		log.Fatalf("Failed to parse own configuration %v error: %v", configFile, err)
 	}
 
-	ocInterceptorAddr := ownConfig.openCensusInterceptorAddressOrDefault()
```
Comment (odeke-em) [inline cmd/ocagent/main.go]: We'll need to keep a mention of OpenCensus interceptor as we'll have more than the OpenCensus interceptor.

**Comment by rakyll** [comment]
All addressed, thanks.

**Pull request merged**